app/models/reimbursement/expense/mileage.rb

```diff
@@ -34,7 +34,7 @@ module Reimbursement
   class Expense
     class Mileage < ::Reimbursement::Expense
       def rate
-        event.plan.mileage_rate(created_at || Date.today) # rate may be used on creation, however, created_at isn't set at that point.
+        (event&.plan || Event::Plan::Standard).mileage_rate(created_at || Date.today) # rate may be used on creation, however, created_at isn't set at that point.
```

Comment:
I think `mileage_rate` is defined on the instance, and not the class. Other than that, this looks good. Thanks!

```suggestion
        (event&.plan || Event::Plan::Standard.new).mileage_rate(created_at || Date.today) # rate may be used on creation, however, created_at isn't set at that point.
```